product.py

```diff
@@ -381,6 +381,15 @@ def fetchIfaceInfoFromNetworkdbAsDict(bridge, iface=None):
             sched_gran = next((x for x in xen_args if x.startswith('sched-gran=')), None)
             if sched_gran:
                 results['host-config']['sched-gran'] = sched_gran
+
+            # Subset of hypervisor arguments
+            kernel_args = boot_config.menu[boot_config.default].getKernelArgs()
+
+            #   - xen-pciback.hide
+            pciback = next((x for x in kernel_args if x.startswith('xen-pciback.hide=')), None)
+            if pciback:
+                results['host-config']['xen-pciback.hide'] = pciback
+
```

Comment:
This only extracts a single `xen-pciback.hide` param, it may make sense to preserve all of them ?

Reply:
the kernel modules just get a single pointer to the parameter